interface/main/messages/js/reminder_appts.js

```diff
@@ -383,16 +383,36 @@ $(function () {
         var url = "save.php";
         top.restoreSession();
         $.ajax({
-            type: 'POST',
-            url: url,
-            data: formData,
-            action: 'save_prefs'
-        }).done(function (result) {
+                   type: 'POST',
+                   url: url,
+                   data: formData,
+                   action: 'save_prefs'
+               }).done(function (result) {
             $("#div_response").html('<span style="color:red;">' + xljs1 + '.</span>');
```

Comment:
Please keep the same indent, and `color: red` can replace to the Bootstrap `text-danger`.